src/main/java/appeng/util/Platform.java

```diff
@@ -1183,7 +1189,7 @@ public static long nanoTime()
 			if( itemToAdd < input.getStackSize() )
 			{
 				final long original = input.getStackSize();
-				final StackType split = (StackType) input.copy();
+				final T split = (T) input.copy();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "T". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1905)